src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -57,7 +60,14 @@ class SelectListRenderer implements IRenderer<ISelectOptionItem, ISelectListTemp
 		const optionDisabled = (<ISelectOptionItem>element).optionDisabled;
 
 		data.optionText.textContent = optionText;
-		data.root.setAttribute('aria-label', nls.localize('selectAriaOption', "{0}", optionText));
+		data.root.setAttribute('aria-label', nls.localize('selectAriaOption', "{0}", optionText) + ',.');
```

Comment:
Hey @cleidigh why the `,.` here?

Reply:
both , and .  will cause most screen readers to add a pause
in this case I found it really important to separate with a pause the options from the detailed description

with a short option like 'off'  it was almost completely missed by the subsequent description.

I was planning to add this to the other controls unless you have an objection

Reply:
Sure, that makes sense. Could you add a comment in the code that explains that?

Reply:
sure, I'll do it along with the gear icon keyboard shortcut readout
...and comment